stories/src/ui/StepByStepNav.stories.mdx

```diff
@@ -0,0 +1,62 @@
+import styled from 'styled-components'
+import { Meta, Story, Canvas } from '@storybook/addon-docs'
+import { StepByStepNav } from '@amsterdam/asc-ui'
+
+export const Decorator = styled.div``
+
+<Meta
+  title="UI/StepByStepNav"
+  component={StepByStepNav}
+  decorators={[
+    (StoryComponent) => (
+      <Decorator>
+        <StoryComponent />
+      </Decorator>
+    ),
+  ]}
+  args={{
+    steps: [
+      { label: 'item #1' },
```

Comment:
I'm refraining from using stubs for now; none of the tests have stubs and in my opinion it would take a core maintainers decision to start using those. Also, there is just one array of small data; doesnt' call for using stubs or fixtures imho.